EOFTests/eof_validation/EOF1_truncated_section.json

```diff
@@ -0,0 +1,49 @@
+{
+  "EOF1_truncated_section": {
+    "vectors": {
+      "EOF1_truncated_section_0": {
+        "code": "0xef0001010004020001000204000000",
+        "results": {
+          "Prague": {
+            "exception": "EOF_InvalidSectionBodiesSize",
+            "result": false
+          }
+        }
+      },
+      "EOF1_truncated_section_1": {
+        "code": "0xef0001010004020001000204000000008000",
+        "results": {
+          "Prague": {
+            "exception": "EOF_InvalidSectionBodiesSize",
+            "result": false
+          }
+        }
+      },
+      "EOF1_truncated_section_2": {
+        "code": "0xef000101000402000100020400000000800000fe",
+        "results": {
+          "Prague": {
+            "exception": "EOF_InvalidSectionBodiesSize",
+            "result": false
+          }
+        }
+      },
+      "EOF1_truncated_section_3": {
```

Comment:
Besu tags this one as failed `Incomplete data section`

https://github.com/ethereum/EIPs/blob/8f43e4cb0fd2fd4385d680e43f7b28e6a9e142ad/EIPS/eip-3540.md#container-validation-rules. - rule 4 state that the size _must_ match.  

I am presuming all of these are deployed or deployable, so the data section limitations in https://github.com/ipsilon/eof/blob/main/spec/eof.md#data-section-lifecycle do not apply.